R/supervised-model-deployment.R

```diff
@@ -457,6 +459,64 @@ SupervisedModelDeployment <- R6Class("SupervisedModelDeployment",
     deploy = function() {
     },
     
+    # Get and write model metadata
+    getMetadata = function() {
```

Comment:
getMetaData. Or is metadata one word? Maybe you could ameliorate by calling this data_for_console_output_without_PHI

Reply:
Ugh. I'm down to resist the move to underscores if others are too. How much work can it be to specify that in our lintr. I think metadata is one word.